themes/sdss_subtheme/src/scss/theme/_paragraphs.scss

```diff
@@ -0,0 +1,14 @@
+@charset "UTF-8";
```

Comment:
These styles should be in a different location. The `/theme` SCSS shouldn't have any structure in it. The original styling for this comes from:

[modules/stanford_profile_styles/lib/scss/component/node/stanford_page.scss](https://github.com/SU-SWS/stanford_profile/blob/c1f48771f5f8a25562fa4641feb18c967fc22a82/modules/stanford_profile_styles/lib/scss/component/node/stanford_page.scss)

I would create a `node` directory in `sdss_subtheme/src/scss/components/` and then create and put these styles in a `_stanford_page.scss` file.